t/55utf8_jp.t

```diff
@@ -11,6 +11,11 @@ require "lib.pl";
 
 my $dbh = DbiTestConnect($test_dsn, $test_user, $test_password, { mysql_enable_utf8 => 1, PrintError => 1, RaiseError => 1 });
 
+if (!MinimumVersion($dbh, '5.5')) {
+    plan skip_all =>
+        "SKIP TEST: You must have MySQL version 5.5 or greater for this test to run";
```

Comment:
But we have already check if `lc_messages` messages are available. This test uses `lc_messages` and not particular MySQL version. So test plan should check for existence of `lc_messages` and not MySQL version if it is possible. And this test is there.